src/api/firebase.js

```diff
@@ -165,7 +165,9 @@ export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	const listCollectionRef = collection(db, listPath, 'items');
 	// TODO: Replace this call to console.log with the appropriate
 	// Firebase function, so this information is sent to your database!
```

Comment:
```suggestion
```
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( non-blocking ):** Feel free to delete this comment!